addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -656,6 +672,23 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
             this.resultsChart.setShowAnswers(showAnswers);
             this.resultsChart.updateChart();
         }
+    },
+    /**
+     * Updates the tooltip for current page (on right arrow icon for 'Next' content).
+     * this method will be called on Clicking of Next and Previous Arrow to show the
+     * tooltip for the Next Content.
+     */
+    _updateNextPageTooltip() {
+        let screenToDisplay = this._getNextScreen();
+        if (this.currentScreen == 'startScreen') {
```

Comment:
linting: should be "==="

If you have a correctly configured linter, it should warn you about this.